reckon-core/src/main/groovy/org/ajoberstar/reckon/core/GitVcs.java

```diff
@@ -0,0 +1,215 @@
+package org.ajoberstar.reckon.core;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.zafarkhaja.semver.ParseException;
+import com.github.zafarkhaja.semver.Version;
+import org.eclipse.jgit.lib.*;
+import org.eclipse.jgit.errors.*;
+import org.eclipse.jgit.revwalk.*;
+import org.eclipse.jgit.revwalk.filter.*;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import java.util.stream.Collectors;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this duplicated import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUselessImportCheck)